.gitignore

```diff
@@ -1,3 +1,6 @@
+# Pycharm
+.idea/
```

Comment:
In general it's a good practice to keep unrelated changes in separate PRs, even if they are small changes like this.